builder/builder-next/builder.go

```diff
@@ -313,10 +313,24 @@ func (b *Builder) Build(ctx context.Context, opt backend.BuildConfig) (*builder.
 	}
 	frontendAttrs["add-hosts"] = extraHosts
 
+	exporterName := ""
 	exporterAttrs := map[string]string{}
 
-	if len(opt.Options.Tags) > 0 {
-		exporterAttrs["name"] = strings.Join(opt.Options.Tags, ",")
+	if len(opt.Options.Outputs) > 1 {
+		return nil, errors.Errorf("multiple outputs not supported")
+	} else if len(opt.Options.Outputs) == 0 {
+		exporterName = "moby"
+	} else {
+		if opt.Options.Outputs[0].Type != "cacheonly" {
```

Comment:
what is cacheonly? where is its implementation?

Reply:
There is no "cacheonly" implementation. This is used as a way to trigger skipping the exporters because in moby API zero value means "moby" exporter.

Reply:
could you add that to the code comment?